proposal/color-4-new-spaces-js.d.ts.md

```diff
@@ -218,68 +285,190 @@ colors as best they can, but it may be necessary in some cases.
 toGamut(): SassColor;
 ```
 
-#### `changeChannels`
-
-Returns a new SassColor as the result of changing some of [`internal`]'s
-channels. The `space` value defaults to the `space` of [`internal`], and any
-combination of channels in that space may be changed. Throws an error if any
-`channel` is not present in `space`.
-
-* If `space` is not defined, let `space` be the value of [`space(internal)`].
+#### `change`
 
-* Let `arguments` be the key value pairs in `channels` as keyword arguments.
+Replace the definition of [color.change] with the following:
+
+[color.change]: ../spec/js-api/value/color.d.ts.md#change
 
-* Return the value of the result of [`change(internal, ...arguments)`].
-
-[`space(internal)`]: #space
-
-[`change(internal, ...arguments)`]: ./color-4-new-spaces.md#colorchange
+This algorithm takes a JavaScript object `options` and returns a new SassColor
+as the result of changing some of [`internal`]'s components.
+
+The `space` value defaults to the `space` of [`internal`], and any combination
+of channels and alpha in that space may be changed.
```

Comment:
"any combination ... may be changed" in a spec suggests that it's up to the implementer what gets changed and when, which isn't accurate.